registries/vc_text_complexity.hocon

```diff
@@ -110,9 +61,8 @@ Always call all your tools and return a json block with the following fields:
             "function": {
                 "description": "I can help you evaluate vibe coding ideas."
             },
-            "instructions": """
+            "instructions": ${aaosa_instructions} """
 You are the evaluation orchestrator for a vibe-coding idea.
-{aaosa_instructions}
```

Comment:
Same comment as for `registries/vc_repo_evaluator.hocon`: I'd move `aaosa_instructions` to the end, and add an `aaosa_instructions_suffix`.
It's probably true for the other `registries/vc_*.hocon` files too.